docs/core/toc.yml

```diff
@@ -238,16 +238,17 @@
   items:
   - name: Overview
     href: tools/index.md
-  - name: Tools
+  - name: Global and local tools
     expanded: true
     items:
-    - name: Global tools
-      expanded: true
-      items:
-      - name: Overview
-        href: tools/global-tools.md
-      - name: Create a global tool
-        href: tools/global-tools-how-to-create.md
+    - name: Overview
+      href: tools/global-tools.md
+    - name: Create a tool
+      href: tools/global-tools-how-to-create.md
+    - name: Install and use a global tool
```

Comment:
Should we add a Tutorials node here? By looking at the TOC right now it looks like I'd find the instructions there and not a tutorial

Reply:
I did that after Genevieve made a similar comment.